extensions/skyportal/static/js/components/ZTFAlert.jsx

```diff
@@ -0,0 +1,476 @@
+// import React from 'react';
+// import React, {useEffect, useState, Suspense} from 'react';
+import React, {useEffect, useState, Suspense} from 'react';
+import {useDispatch, useSelector} from 'react-redux';
+
+import styles from "./ZTFAlert.css";
+
+import Button from '@material-ui/core/Button';
+import SaveIcon from '@material-ui/icons/Save';
+import PropTypes from 'prop-types';
+import {lighten, withStyles, makeStyles} from '@material-ui/core/styles';
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableContainer from '@material-ui/core/TableContainer';
+import TableHead from '@material-ui/core/TableHead';
+import TablePagination from '@material-ui/core/TablePagination';
+import TableRow from '@material-ui/core/TableRow';
+import TableSortLabel from '@material-ui/core/TableSortLabel';
+import Paper from '@material-ui/core/Paper';
+import Grid from '@material-ui/core/Grid';
+
+import Responsive from "./Responsive";
+import FoldBox from "./FoldBox";
+
+// import Moment from 'react-moment';
+
+// Import action creators from `static/js/ducks/alert.js`
+import * as Actions from '../ducks/alert';
+import {FETCH_ALERT_ERROR, FETCH_ALERT_FAIL, FETCH_ALERT_OK} from "../ducks/alert";
+
+const VegaPlot = React.lazy(() => import('./VegaPlotZTFAlert'));
+
+
+const StyledTableCell = withStyles((theme) => ({
+  head: {
+    // backgroundColor: theme.palette.common.black,
+    // backgroundColor: "#111",
+    // color: theme.palette.common.white,
+    // color: "#f0f0f0",
+  },
+  body: {
+    fontSize: 14,
+  },
+}))(TableCell);
+
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    width: '100%',
+  },
+  container: {
+    maxHeight: 440,
+  },
+  whitish: {
+    color: "#f0f0f0"
+  },
+  visuallyHidden: {
+    border: 0,
+    clip: 'rect(0 0 0 0)',
+    height: 1,
+    margin: -1,
+    overflow: 'hidden',
+    padding: 0,
+    position: 'absolute',
+    top: 20,
+    width: 1,
+  },
+  margin_bottom: {
+    "margin-bottom": "2em",
+  },
+  margin_left: {
+    "margin-left": "2em",
+  },
+  image: {
+    padding: theme.spacing(1),
+    textAlign: 'center',
+    color: theme.palette.text.secondary,
+  },
+  paper: {
+    padding: theme.spacing(2),
+    textAlign: 'center',
+    color: theme.palette.text.secondary,
+  },
+// table: {
+//     minWidth: 650,
+// },
+}));
+
+function createRows(id, jd, fid, mag, emag, rb, drb, isdiffpos, programid, alert_actions) {
```

Comment:
this just bundles the arguments into an object? is this worth defining as a function rather than doing in-line?